docs: Refactor condition messages and reasons (#1099)
kyma-project/telemetry-manager: docs/user/02-logs.md

### Max Amount of Pipelines

The maximum amount of LogPipelines is 5.

## Troubleshooting

### Logs Not Arriving at the Destination

>    Cause: The backend is not reachable or wrong authentication credentials are used.

Reviewer: Maybe let's add the aspect of using a wrong endpoint. Saying "not reachable" sounds like a temporary outage.

   Remedy: Investigate the cause with the following steps:
   1. Check the `telemetry-fluent-bit` Pods for error logs by calling `kubectl logs -n kyma-system {POD_NAME}`.

### Influx Capacity Reaching Its Limit

>   Cause: The backend sizing is inadequate

Reviewer 2: ```suggestion
  Cause: The backend sizing is too small for the data influx.
```

  Remedy:

>   - Option 1: Increase ingestion rate capabilities in your backend (i.e. scale out SAP Cloud Logging or Dynatrace instances)

Reviewer 2: ```suggestion
  - Option 1: Increase ingestion rate capabilities in your backend. For example, scale out the SAP Cloud Logging or Dynatrace instances.
```

>   - Option 1: Increase ingestion rate capabilities in your backend (i.e. scale out SAP Cloud Logging or Dynatrace instances)

Reviewer: Our LogPipeline does not support Dynatrace integration. Let's remove it from this section.

>   - Option 2: Decrease data influx (i.e. re-configure the log pipeline)

Reviewer 2: ```suggestion
  - Option 2: Decrease data influx, that is, reconfigure the log pipeline.
```

>   - Option 2: Decrease data influx (i.e. re-configure the log pipeline)

Reviewer: Or reduce emitted logs by application. E.g., by changing log-level.
